test/crowdsale/WhitelistedCrowdsale.test.js

```diff
@@ -0,0 +1,59 @@
+const { ether } = require('../helpers/ether');
+const shouldFail = require('../helpers/shouldFail');
+
+const BigNumber = web3.BigNumber;
+
+require('chai')
+  .should();
```

Comment:
Shouldn't we be using the new setup helper?

Reply:
Yup, this is from before that was merged.